### ISSUE TITLE
Can you guys Recreate the repository again?

### ISSUE THREAD

**Issue by schwannden** [metaOmic/MetaPath#1]
Because @xiaoguang1988  is the one pushing the repository, so all open source record goes to @xiaoguang1988 instead of @XiangruiZeng , this is a little bit odd XDD

After all, the ability to record contribution to projects is one motivation for open source development. If @XiangruiZeng knows all the step, can he delete the repository and push it with his account configuration?

**Comment by xiaoguang1988** [comment]
Yes, I suggest Taylor to delete my pushed package and he can upload himself again.
Or I can delete it when Taylor is ready if Taylor don't have the right to do so.

**Comment by schwannden** [comment]
So the thing is, you need to keep a local copy of the repository, and then you will delete the repository on Github, then you create a repository (Taylor can't create a repository because he is not the owner of the organization "metaOmic") on Github, then **Taylor** push his local repository to Gtihub (Taylor can push, because he is admin of team "Meta Path").
